tutorials/btp-integration-suite-execute-batch-payload/btp-integration-suite-execute-batch-payload.md

```diff
@@ -0,0 +1,377 @@
+description: Learn how to connect, interact, and perform batch operations on your database via JDBC receiver adapter.
+keywords: tutorial
+auto_validation: true
+time: 40
+tags: [ tutorial>intermediate, software-product>sap-integration-suite]
+primary_tag: software-product>sap-business-technology-platform
+parser: v2
+---
+
+## You will learn
+-	To connect Java Database Connectivity (JDBC) receiver adapter with your database (on the receiver system) and send, receive, or update data on the database.
+-	To interact with your database using SQL queries and XML.
+-	To perform operations on database using Batch Payload functionality.
+
+
+## Prerequisites
+
+ - You have enabled the Cloud Integration capability of [SAP Integration Suite](http://bit.ly/3VmWOOZ).
+ - You have configured your [JDBC adapter](https://blogs.sap.com/2018/10/31/configuring-jdbc-adapter-a-step-by-step-guide/).
+
+
+## Intro
+The data that you are transferring (sending or receiving) is called payload. The payload sent to the JDBC adapter can be modified. You can modify multiple records in a single payload using either INSERT, UPDATE, or DELETE modes. This can be achieved using Batch Payload functionality.
+
+There are two types of Batch operations:
+
+-	Atomic: This mode considers each batch operation as a single unit. It updates the whole batch operation successfully or reverts the entire operation to its initial state if anything in the batch operation fails.
+
+-	Non-Atomic- The operation in this mode varies depending upon the database and its driver. Generally, it updates all the successfully executed records and throws an exception if anything fails. It does not revert the failed records to its initial state.
+
+Following the steps below, you will be able to setup a sample database, create an integration flow, upload drivers, and data sources required for JDBC receiver adapter. Then, you can create a payload, execute, and benefit from Batch Payload functionality.
+
+
+### Choose your Database
+Based on your requirements, choose a database you wish to work on. JDBC receiver adapters supports more than ten databases. To know more, see [JDBC: Supported Databases](https://help.sap.com/docs/CLOUD_INTEGRATION/368c481cd6954bdfa5d0435479fd4eaf/88be64412f1b46d684dfba11f2767c5b.html?locale=en-US&version=Cloud).
+
+If you're connecting to on-premise database, then you must validate the connection in the **Cloud Connector**. For more information, see [JDBC: Access Control](https://help.sap.com/docs/CP_CONNECTIVITY/cca91383641e40ffbe03bdc78f00f681/f42fe4471d6a4a5fb09b7f3bb83c66a4.html?version=Cloud&locale=en-US).
+
+
+### Create data source and upload driver
+JDBC adapter supports third party and in-house databases on Neo and Cloud Foundry Business Technology environments. Before you establish the connection to these databases, or setup an integration flow, you need to upload drivers (applicable for HANA, ASE, PostgreSQL only) and add Data Sources.
```

Comment:
Please fix grammar -> the setup, to set up

Reply:
reframed